CHANGELOG.md

```diff
@@ -63,6 +63,7 @@
 - Fix dropped "n" characters on some platforms in definition file stored as part
   of SIF metadata.
 - Pass STDIN to `--oci` containers correctly, to fix piping input to a container.
+- Fix memory usage calculation during singularity compilation on RaspberryPi.
```

Comment:
```suggestion
- Fix compilation on 32-bit systems.
```

Let's change the original message here. It's not really important exectly where in the code the issue is. Also, it's for 32-bit systems in general, rather than just Raspberry Pi.